### ISSUE TITLE
Create a React component UI library

### ISSUE THREAD

**Issue by dvd-schwrtz** [graphprotocol/mission-control-curator#211]
Add a library built in React that makes it really easy for others to use these components to obtain the subgraph data for all or a subset of categories and get the categorized data. The UI library can allow for custom color schemes, custom stylings or default stylings.

**Comment by yanivtal** [comment]
Yes! Great idea. We actually built Everest with ThemeUI so it would be simple to reuse. Some could create a separate component library out of this https://github.com/graphprotocol/everest/tree/master/ui/src/components.
